apps/notifications/src/standalone/index.js

```diff
@@ -1,12 +1,21 @@
-import '@babel/polyfill';
+/**
+ * External dependencies
+ */
+import 'boot/polyfills';
```

Comment:
Does this import really resolve correctly for a standalone Notifications app that's not part of Calypso app? These imports shouldn't automatically resolve to `client/`

Reply:
You're right, this does not work! I'll go ahead and turn the polyfills into a package, as you suggested 👍

Reply:
A npm package might be more challenging than I originally thought: the env preset needs to transform the `core-js` imports inside it, differently for each build target.

Reply:
Yes, you're right 😞 I'll use just the bare `core-js` polyfills (+regenerator) here.